apps/alb_monitor/test/alb_monitor/monitor_test.exs

```diff
@@ -11,18 +11,18 @@ defmodule ALBMonitor.MonitorTest do
     mock_instance_ip("10.0.0.2")
     mock_health_response([{"10.0.0.1", "healthy"}, {"10.0.0.2", "draining"}])
 
-    start!()
+    ref = start!()
```

Comment:
question: what does this `start!()` function do? I can't find where it's defined.